src/app/admin/clients/page.tsx

```
@@ -363,69 +373,42 @@ export default function AdminClientsPage() {
+function ClientsPage() {
+  // Define the missing dialog state
+  const [showAssignDialog, setShowAssignDialog] = useState(false)
+
+  // ... other code
+
+                    setShowAssignDialog(true)
+
+  // ... rest of the component
+}
+
+export default ClientsPage
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)